GameOfLife/Calculations/Rules.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GameOfLife
+{
+    class Rules
+    {
+        public int[,] CellCalculation(int[,] GameField)
+        {
+            int[,] tempArr = (int[,])GameField.Clone();
+            int lenght = GameField.GetLength(0);
+            int width = GameField.GetLength(1);
+
+            for (int i = 0; i < lenght; i++)
+            {
+                for (int j = 0; j < width; j++)     // set copied same size array elements to 0
+                {
+                    tempArr[i, j] = 0;
+                }
+            }
+
+
+            for (int i = 0; i < lenght ; i++)
+            {
+                for (int j = 0; j < width ; j++)
+                {
+                    int liveCells = 0; // Livecell neighbour count for each cell
+
+                    for (int x = -1; x < 2; x++)
```

Comment:
These two looping fors could probably be moved to a separate method, here in this same file, and then just the method called here to get the alive neighbor count

Reply:
Moved in separate method for finding liveNeighbours